src/helpers/ball-data.js

```diff
@@ -0,0 +1,735 @@
+module.exports = {
+  __explanation__: `
```

Comment:
If this is a JS file, we can probably put the explanation in a comment. (I think the `__explanation__` key is from when this file was JSON.)